.moban.yaml

```diff
@@ -29,6 +29,7 @@ targets:
   - test-requirements.txt: bears-test-requirements.txt.jj2
   - bears/VERSION: VERSION.jj2
   - Gemfile: Gemfile.jj2
+  - description: description.jj2
```

Comment:
You should also commit the generated `DESCRIPTION` file after changing the key according to the comment by sangamcse at https://github.com/coala/coala-bears/pull/2731#pullrequestreview-168534667

Reply:
Here is a DESCRIPTION file I generated a long time ago : https://github.com/coala/coala-bears/pull/1386/files#diff-35ba4a2677442e210c23a00a5601aba3

It didnt have any versions, and it worked OK without them.

Reply:
Or you can add versions currently used by CI,  lintr 1.0.2 and formatR 1.5 , and put them into bears/r/RLintBear.py and `FormatRBear.py` respectively.